src/client.ts

```diff
@@ -69,25 +79,57 @@ export default class UnleashClient extends EventEmitter {
                     `Malformed feature, strategies not an array, is a ${typeof feature.strategies}`,
                 ),
             );
-            return false;
+            return { enabled: false };
```

Comment:
Why does this method not simply return the `boolean` value?

Reply:
So it can be used by getVariant.

Reply:
Changed the wrapper :)